index.html

```diff
@@ -0,0 +1,278 @@
+                Naturally creative and an adept problem-solver with proficiency
+                in a wide range of coding languages, including:
+              </p>
+              <ul>
+                <li><strong>HTML</strong> including semantic refactoring</li>
+                <li>
+                  <strong>CSS</strong> including flexbox - media queries -
+                  variables
+                </li>
+                <li>
+                  <strong>JavaScript</strong> including loops - arrays -
+                  conditional statements
+                </li>
+              </ul>
+              <p>
+                You can find a copy of my CV
+                <a
+                  href="assets/files/cvCherelleSimpson.pdf"
+                  id="cv"
+                  target="_blank"
+                  >here</a
+                >.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!-- second section containing app image, title, tech and link -->
+      <section class="portfolio-container">
+        <h2 class="section-title" id="portfolio-title">Portfolio</h2>
+        <div class="portfolio-flex-container">
+          <!-- main image -->
+          <div class="portfolio-main-container">
+            <div>
+              <a
+                href="https://c-sim.github.io/semantic-html-refactor/"
+                target="_blank"
+                ><img
+                  class="portfolio-img1"
+                  src="./assets/images/semantic-html-refactor.png"
+                  alt="semantic-html-refactor screenshot"
+              /></a>
+            </div>
+            <div class="portfolio-name">
+              <h3>Semantic HTML Refactor</h3>
+              <p>HTML - CSS</p>
+              <div>
+                <a
+                  href="https://github.com/C-Sim/semantic-html-refactor"
+                  target="_blank"
+                >
+                  <i class="fa-brands fa-github" alt="GitHub"></i
+                ></a>
+              </div>
+            </div>
+          </div>
+
+          <!-- sub images -->
+          <div class="portfolio-sub-container">
+            <div class="project-card">
+              <div>
+                <a href="https://c-sim.github.io/portfolio/" target="_blank">
+                  <img
+                    class="portfolio-img2"
+                    src="./assets/images/Portfolio.png"
+                    alt="Portfolio screenshot"
+                /></a>
+              </div>
+              <div class="portfolio-name">
+                <h3>Personal Portfolio</h3>
+                <p>HTML - CSS</p>
+                <div>
+                  <a href="https://github.com/C-Sim/portfolio" target="_blank">
+                    <i class="fa-brands fa-github" alt="GitHub"></i
+                  ></a>
+                </div>
+              </div>
+            </div>
+            <div class="project-card">
```

Comment:
It would be great to leave a blank line between each of these cards to break up your code a little. At the moment it's not as easy to read as it would be if there was more spacing.

Reply:
Great shout, thank you. I was even getting myself confused at times when working through the cards so this is a useful learning...if I'm having trouble deciphering each part, others will too :)